webui/module/Restore/src/Restore/Controller/RestoreController.php

```diff
@@ -596,7 +596,7 @@ public function updateRestoreParams($restore_source_clients, $restore_target_cli
             $this->restore_params['restoreclient'] = $restore_target_clients[0]['name'];
         }
         if($restorejobresources && (count($restorejobresources) == 1)) {
-            $this->restore_params['restorejob'] = array_key_first($restorejobresources);
+            $this->restore_params['restorejob'] = array_keys($restorejobresources)[0];
```

Comment:
Maybe we should add a comment explaining that we are not using `array_key_first` for php 7.2 support.

This should make it easier to change once we dont need to support it anymor.

Reply:
done